src/fr/naruse/spleef/spleef/type/Spleef.java

```diff
@@ -430,7 +431,7 @@ public void makeLose(Player p) {
     }
 
     public void checkWin() {
-        if(playerInGame.size() == 1){
+        if(playerInGame.size() == 1 && currentStatus == GameStatus.GAME){
```

Comment:
added check if they are actully ingame

Reply:
Gonna check that tonight. And fix more if needed, then update. Thanks for help :)